Source/Android/app/build.gradle

```diff
@@ -65,7 +65,7 @@ android {
         externalNativeBuild {
             cmake {
                 arguments "-DANDROID_STL=c++_static", "-DCMAKE_BUILD_TYPE=RelWithDebInfo" // , "-DENABLE_GENERIC=ON"
-                abiFilters "arm64-v8a", "x86_64" //, "armeabi-v7a", "x86"
+                abiFilters "arm64-v8a", "armeabi-v7a" //,"x86_64", "x86"
```

Comment:
Whoops, I'll fix that ASAP. I have to keep changing these in order to get the project to run on my phone.